circuit_knitting/cutting/cutting_decomposition.py

```diff
@@ -13,6 +13,7 @@
 
 from __future__ import annotations
 
+
```

Comment:
```suggestion
```
will fully remove this file from the diff

Reply:
Should be done now...

Reply:
I still see it in the overall diff at https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/471/files#diff-fcbd1b39891440c3ebf98536ee92ccbf9f7d84ed89db13422f95362ae8dbe3cb but not a big deal.